paper/paper.md

```diff
@@ -36,19 +36,19 @@ bibliography: paper.bib
 ## Summary
 
 The advent of commercial cloud services has provided researchers with the benefits of flexible and scalable computing and storage resources.
-However, they present a challenge for organisations that require predictability in their expenditure, as the cost control mechanisms are not always well suited to large numbers of research projects with varied budgetary constraints.
-
-In response, we have developed [RCTab](https://rctab.readthedocs.io/) (**R**esearch **C**omputing **Tab**les), an open-source system for automating the management of [subscriptions](https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-setup-guide/organize-resources#management-levels-and-hierarchy) on Azure, Microsoft’s cloud computing platform.
-RCTab automates the mundane management tasks of monitoring usage, alerting stakeholders of impending overspend and stopping subscriptions once they are over-budget.
-This saves administrative time, reduces the risk of excessive spending and provides a rich source of data for usage forecasting.
-
+Unfortunately, cloud providers do not always provide the ability to enforce strict budget controls, allowing users to overspend.
+This can present serious challenges for adoption in research organisations that need to centrally disseminate cloud resources to researchers with independent budgets.
```

Comment:
> This can present serious challenges for adoption in research organisations that need to centrally disseminate cloud resources to researchers with independent budgets.

This is such is a good way to phrase it.